Apps/WebClient/src/ClientApp/src/services/restBetaRequestService.ts

```diff
@@ -23,7 +23,7 @@ export class RestBetaRequestService implements IBetaRequestService {
         this.http = http;
     }
 
-    public getRequest(hdid: string): Promise<BetaRequest> {
+    public getRequested(hdid: string): Promise<BetaRequest> {
```

Comment:
naming issue, how about just call it "get" as it is a method of BetaRequestService :-)

Reply:
I dont think get is better, although I am not sure what to name it either :)